modules/admin/pages/body.inc

```diff
@@ -2,284 +2,309 @@
 /*.
     require_module 'standard';
 .*/
-require_once($PAGESDIR.'/base/leftnav.inc');
-require_once($FUNCTIONDIR.'/authentication.inc');
-require_once(__DIR__."/../functions/admin.inc");
-require_once($FUNCTIONDIR.'/users.inc');
+
+require_once $PAGESDIR . '/base/leftnav.inc';
+require_once $FUNCTIONDIR . '/authentication.inc';
+require_once __DIR__ . "/../functions/admin.inc";
+require_once $FUNCTIONDIR . '/users.inc';
+
+const FIRST_NAME = 'firstName';
+const LAST_NAME = 'lastName';
+const EMAIL = 'email';
+const ID = 'ID';
+const FIELD = 'Field';
 ?>
-<div id="main_content"  class="UI-cell-top UI-maincontent">
-<?php
-echo "<h2 class='UI-sectionbar'>Welcome ".$_SESSION['preferredName']."!</h2>\n";
-/*
-if( isset($_REQUEST['DoFeed'])) {
-echo "<hr><h3>Starting DeFeed Routine:</h3>\n";
-$lottery = search_definedFields('Lottery Status');
-$status = array_search('Entered', $_SESSION['definedFields']['Lottery Status']);
-$bedPref = search_definedFields('Bed Preference');
-$noBed = array_search('No Preference', $_SESSION['definedFields']['Bed Preference']);
-$kingBed = array_search('One King', $_SESSION['definedFields']['Bed Preference']);
-$queenBed = array_search('Two Queens', $_SESSION['definedFields']['Bed Preference']);
-$hotelPri = search_definedFields('Hotel Qualifier');
-$volPri = array_search('Volunteer Qualifier', $_SESSION['definedFields']['Hotel Qualifier']);
-$adaPri = array_search('ADA Wheelchair', $_SESSION['definedFields']['Hotel Qualifier']);
-$adoPri = array_search('ADA Other', $_SESSION['definedFields']['Hotel Qualifier']);
+<div id="main_content" class="UI-cell-top UI-maincontent">
+    <?php
+    echo "<h2 class='UI-sectionbar'>Welcome " . $_SESSION['preferredName'] . "!</h2>\n";
+    /*
+     * //TODO: Is this code throw away?  There is quite a lot here...
```

Comment:
@AricStewart, can I delete that?

Reply:
Delete the TODO? yes, that is fine.

Reply:
I added the todo to mark it.  I'm referring to the commented code under the TODO.  There's a large amount there.

Reply:
It is inside the comment block, and looks to have to do with the hotel lottery so as far as I am concerned it can all go.

Reply:
@mshappe / @tskeeley, any rebrutals?